wp-content/themes/pub/wporg-learn-2020/archive-lesson-plan.php

```diff
@@ -45,25 +45,23 @@
 			'orderby'    => 'id',
 		) );
 		?>
-		<div class="row lp-taxonomy categories">
+		<div class="lp-taxonomy">
 			<div class="card-grid card-grid_4">
-				<h2 class="h4 lp-taxonomy-header"><?php echo esc_html__( 'Topic', 'wporg-learn' ); ?></h2>
-				<div class="lp-taxonomy-description"><?php echo esc_html__( 'Browse lesson plans by their high-level topic.', 'wporg-learn' ); ?></div>
+				<header class="card-grid-header">
```

Comment:
These non `card` elements within the `card-grid` cause responsive issues. Wrapping them with this div means we can control their behaviour and force them to always be full width, no matter what width the cards are set to